about/introduction.md

```diff
@@ -1,9 +1,10 @@
+# Introduction
 
-xmake is a lightweight cross-platform build utility based on Lua. It uses xmake.lua to maintain project builds. Compared with makefile/CMakeLists.txt, the configuration syntax is more concise and intuitive. It is very friendly to novices and can quickly get started in a short time. Let users focus more on actual project development.
+Xmake is a lightweight cross-platform build utility based on Lua. It a Lua script to maintain project builds. Compared with Makefile/CMakeLists.txt, the configuration syntax is much concise and intuitive. As such, it's friendly to novices while still maintaining the flexibly required in a build system. With Xmake, you can focus on your project instead of the build.
```

Comment:
> It a Lua script to maintain

It is not just a lua script, it is a complete binary program. But it uses xmake.lua to maintain the project.

Reply:
Ah, sorry. Didn't come across right then. I'll resolve it.